requirements.txt

```diff
@@ -1,2 +1,3 @@
 canonicalwebteam.flask-base==0.3.3
-gunicorn[gevent]
+canonicalwebteam.discourse-docs==0.6.5
+gunicorn[gevent]
```

Comment:
This is possibly a bit over-cautious, but would you mind placing `gunicorn[gevent]` directly below `flask-base`? If `pip` were a decent package manager it wouldn't matter, but as things are there's a possibility that a sub-dependency from `discourse-docs` could lead to `gevent` not installing its preferred version of a dependency, which could be problematic.